Makefile

```diff
@@ -20,13 +20,13 @@
 
 # read README.external for more information about the configuration
 # B.A.T.M.A.N. debugging:
-export CONFIG_BATMAN_ADV_DEBUG=n
+export CONFIG_BATMAN_ADV_LEGACY_DEBUG=n
```

Comment:
As this is always built out-of-tree, I don't see a point in renaming these config symbols.